### ISSUE TITLE
[ FF7 ] Odd behaviour when pressing CTRL key

### ISSUE THREAD

**Issue by LaZar00** [julianxhokaxhiu/FFNx#507]
**Describe the bug**
In the PC version of FF7 you can press CTRL+Q at the same time to show a window for exit the game.
In this case, this window is shown also pressing 'CTRL', releasing it, and then after pressing 'Q' key.

1. Load a game
2. Press CTRL key (release it, so no keys pressed in keyboard)
3. Press Q key (the exit window is shown)

The exit window ONLY should be shown when pressing CTRL+Q at the same time.

I tested this running FF7.exe directly from the game folder with FFNx driver. So, no mods or other apps affect this behaviour.

**Comment by LaZar00** [comment]
BTW, this behaviour is a bit annoying. There are people that is using the keys 'Q' and 'E' as L1/L2 triggers in keyboards (as me, and I've seen other people reporting the same issue in discord). So, if you use CTRL+Up to increase Speedhack, you know that the next time you press 'Q' key will appear the exit window.

**Comment by julianxhokaxhiu** [comment]
@LaZar00 is this still happening? I was never able to reproduce it here. If it is no more happening, I'd close this one. Thanks!

**Comment by LaZar00** [comment]
@julianxhokaxhiu Hi. Yep, I just updated to FFNx 1.15.0.6 to check it again with the new version (btw, THE LOGO IS COOL!) :) but it happens the same.
It seems very odd you can not reproduce it. Maybe it is my notebook?
You only need to load a save, push LEFT-CTRL (without any other key), then push 'Q' key for quit. Pushing them individually makes appear the exit window. It should only be shown when the combination of both keys is pressed at the same time.

**Comment by julianxhokaxhiu** [comment]
Thanks for coming back on this one :) And thanks for your kind words, I'm happy you like it!

@CosmosXIII @tangtang95 are you able to replicate this one? Maybe it's just me. I require more feedback on this one.

**Comment by LaZar00** [comment]
This is:

https://user-images.githubusercontent.com/46090031/216828189-f07df9b6-ac92-4c4a-b2aa-4828072905af.mp4

**Comment by LaZar00** [comment]
And my ff7input.cfg:
[ff7input.zip](https://github.com/julianxhokaxhiu/FFNx/files/10611376/ff7input.zip)

**Comment by eve-atum** [comment]
I just replicated on my desktop... 7H 3.0.0.4, FFNX 1.15.0.23

**Comment by tangtang95** [comment]
> Thanks for coming back on this one :) And thanks for your kind words, I'm happy you like it!
> 
> @CosmosXIII @tangtang95 are you able to replicate this one? Maybe it's just me. I require more feedback on this one.

Forgot about this. Anyway, I can replicate it easily

**Comment by zaphod77** [comment]
yeah i 100% can reproduce this too.  once control is pressed, no matter how many other keys you press after, q becomes quit.
